examples/popups/popup_titles_and_format.ipynb

```diff
@@ -1286,19 +1286,14 @@
     "from cartoframes.viz import Layer, popup_element\n",
     "\n",
     "Layer(\n",
-    "    'countries_africa',\n",
+    "    'spain_populated_places',\n",
+    "    basic_style(size=10),\n",
     "    popup_hover=[\n",
-    "        popup_element('name', 'Country', 'population')\n",
+    "        popup_element('name', 'City'),\n",
+    "        popup_element('pop_max', title='Rounded population', format=',.2r')\n",
```

Comment:
LGTM. I'm gonna ping here @manmorjim as he's interested in popup and formatting. It looks like using d3.format on both CF and web-sdk could be nice combination